web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -338,7 +339,7 @@ private String getContextPath(ServletContext servletContext) {
 	private void clearDWRFile(ServletContext servletContext) {
 		String realPath = servletContext.getRealPath("");
 		String absPath = realPath + "/WEB-INF/dwr-modules.xml";
```

Comment:
this should already be replaced with `Paths.get` to avoid using the OS specific path separators `/`